pkg/kwokctl/runtime/cluster.go

```diff
@@ -220,9 +221,9 @@ func (c *Cluster) KubectlInCluster(ctx context.Context, stm exec.IOStreams, args
 }
 
 func (c *Cluster) AuditLogs(ctx context.Context, out io.Writer) error {
-	logs := c.GetLogPath(AuditLogName)
+	logs := filepath.Clean(c.GetLogPath(AuditLogName))
 
-	f, err := os.OpenFile(logs, os.O_RDONLY, 0644)
+	f, err := os.OpenFile(logs, os.O_RDONLY, 0600)
```

Comment:
```suggestion
	f, err := os.OpenFile(logs, os.O_RDONLY, 0640)
```

The same goes for everything else